### DISCUSSION TITLE
Copilot Stuck on waiting for GitHub Authorization

### DISCUSSION THREAD

**Discussion by matquant14** [community/community#29467] [Copilot Conversations]
Using latest release of PyCharm 2022.2 and latest plugin version of Copilot.  Sometimes it fails to initiate the login.  Other times it works, generates the authentication code, but gets stuck on receiving authorization back from GitHub after provide said code.  How can this be fixed?

![image](https://user-images.githubusercontent.com/43152300/184574276-8ef70213-83f1-4ce5-8699-db118386858e.png)

**Comment by nchandra-seg** [comment]
Did anyone fix this?. I am facing the same issue.
